### ISSUE TITLE
i18n guidelines about plurals say "Always include %d in "one"

### ISSUE THREAD

**Issue by antonv6** [commons-app/apps-android-commons#522]
Here's the important part from https://developer.android.com/guide/topics/resources/string-resource.html#Plurals (first example for Plurals section)

```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <plurals name="numberOfSongsAvailable">
        <!--
             As a developer, you should always supply "one" and "other"
             strings. Your translators will know which strings are actually
             needed for their language. Always include %d in "one" because
             translators will need to use %d for languages where "one"
             doesn't mean 1 (as explained above).
          -->
        <item quantity="one">%d song found.</item>
        <item quantity="other">%d songs found.</item>
    </plurals>
</resources>
```

Here's what this project's strings.xml has:

```
  <plurals name="uploads_pending_notification_indicator">
    <item quantity="one">1 file uploading</item>
    <item quantity="other">%d files uploading</item>
  </plurals>
```

I don't know much about android i18n, but this looks contrary to the guidelines.

**Comment by akaita** [comment]
I believe this was fixed in #544 -> 5639be077e943fd4bd7d0859ed10ee94531ecd7c
So this issue could probably be closed.

**Comment by misaochan** [comment]
Agreed, thanks @akaita !
